x-pack/performance/journeys/ecommerce_dashboard.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Journey } from '@kbn/journeys';
+import { subj } from '@kbn/test-subj-selector';
+
+import { ToastsService } from '../services/toasts';
```

Comment:
Do we want to abstract operations like this Toast service ? I had an assumption that we should give the full control to devs who write these journeys. I believe this was the main reason why we didn't wanna choose functional tests to benchmark

Reply:
In what way is this removing control from devs? This toasts service is just a class which is new'd up in the journey and provided to each step via `extendContext()` so that it can have a reference to the page. It's completely optional and just an example of how people can share logic without needing to pass the `page` object to every shared helper (like `waitForVisualizations()`).